app/controllers/metadata/MetadataController.scala

```diff
@@ -84,13 +84,27 @@ class MetadataController @Inject()(
   private def sendEmailIfVerified(notification: Notification)(implicit hc: HeaderCarrier): Future[Boolean] = {
```

Comment:
The statement coverage for the repo will drop by like 0.21%, which isn't the end of the world but if we can unit test the new paths that will likely fix that. I think its probably the case none/case exc that are not covered.

[info] All done. Coverage was stmt=[92.50%] branch=[93.90%] (main)
[info] All done. Coverage was stmt=[92.29%] branch=[93.90%] (MMCA-4297)